kubeproxy/config/manager/manager.yaml

```diff
@@ -37,3 +44,10 @@ spec:
             cpu: 100m
             memory: 20Mi
       terminationGracePeriodSeconds: 10
+      tolerations:
+      - key: "node.kubernetes.io/not-ready"
+        operator: "Exists"
+        effect: "NoSchedule"
+      - key: "node-role.kubernetes.io/master"
+        operator: "Exists"
+        effect: "NoSchedule"
```

Comment:
Same story here -- I think these should be separated into `tolerate-controlplane.patch.yaml`
It's actually possible to create a separate variant for this kind of install, but starting with separate patches is a good step.